Core/Utilities/DErrno.py

```diff
@@ -257,6 +259,7 @@
               EWMSUKN : "Unknown WMS error",
               EWMSJDL : "Invalid job description",
               EWMSRESC : "Job to Reschedule",
+              EWMSBULK : "Invalid bulk submission transaction",
```

Comment:
Do we really want to multiply the number of errors ? I think this is counter productive. Is there no other error in which you can fit yours ? At the very least , I would say that there should be no difference between bulk and non bulk submission.